Countries/FirstScreen/ListViewController.swift

```diff
@@ -15,14 +15,30 @@ class ListViewController: UITableViewController {
     
     var countries: [CountriesApiQuery.Data.Country] = []
     
+    let refrechControll: UIRefreshControl = {
+        let refrechControll = UIRefreshControl()
+        return refrechControll
+    }()
+
+    @objc private func refresh(sender: UIRefreshControl) {
+        loadData()
+        sender.endRefreshing()
+    }
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        setupRefreshController()
```

Comment:
It is not an issue, but I would prefer to set refresh control when you already setup your table view